### DISCUSSION TITLE
代码块主题

### DISCUSSION THREAD

**Discussion by sicl5300** [mirai-mamori/Sakurairo#604] [Q&A] [answered]
请问如何更改hljs代码块主题？默认的黑色有点不和谐。另外，更改代码块主题后，能否做到深色模式和普通模式下使用不同的主题？

**Comment by sicl5300** [comment]
已解决。
昨天眼瞎了没看到切换深色模式后会给body添加dark类，CSS可匹配。

**Comment by KotoriK** [comment, marked as answer]
使用Prism.js进行代码高亮时会自动根据深色模式切换
